.github/actions/install-rust/main.js

```diff
@@ -12,8 +12,11 @@ child_process.execFileSync('rustup', ['set', 'profile', 'minimal']);
 child_process.execFileSync('rustup', ['update', toolchain, '--no-self-update']);
 child_process.execFileSync('rustup', ['default', toolchain]);
 
-// Deny warnings on CI to keep our code warning-free as it lands in-tree
-console.log(`::set-env name=RUSTFLAGS::-D warnings`);
+// Deny warnings on CI to keep our code warning-free as it lands in-tree. Don't
+// do this on nightly though since there's a fair amount of warning churn there.
+if (!toolchain.startsWith('nightly')) {
```

Comment:
Are there toolchains which names start with `nightly` but are not `nightly`? Otherwise we could use `toolchain === 'nightly'` here.

Reply:
Yeah sometimes we'll switch to `nightly-$date` temporarily if a regression sneaks into nightly by accident.

Reply:
I see, thanks (resolving).